### PR TITLE
UIOR-1200: UX Consistency > Search Results > Update HTML page title with search term entered

### PR DESCRIPTION
**Pull request by alisher-epam**
## Purpose

[UIOR-1200](https://folio-org.atlassian.net/browse/UIOR-1200) - UX Consistency > Search Results > Update HTML page title with search term entered
## Approach

https://github.com/folio-org/ui-orders/assets/116072773/b4dd3112-7882-41dd-97d4-fec949257a19
## Pre-Merge Checklist
Before merging this PR, please go through the following list and take appropriate actions.
- [ ] I've added appropriate record to the CHANGELOG.md
- Does this PR meet or exceed the expected quality standards?
  - [ ] Code coverage on new code is 80% or greater
  - [ ] Duplications on new code is 3% or less
  - [ ] There are no major code smells or security issues
- Does this introduce breaking changes?
  - [ ] If any API-related changes - okapi interfaces and permissions are reviewed/changed correspondingly
  - [ ] There are no breaking changes in this PR.

If there are breaking changes, please **STOP** and consider the following:

- What other modules will these changes impact?
- Do JIRAs exist to update the impacted modules?
  - [ ] If not, please create them
  - [ ] Do they contain the appropriate level of detail?  Which endpoints/schemas changed, etc.
  - [ ] Do they have all they appropriate links to blocked/related issues?
- Are the JIRAs under active development?
  - [ ] If not, contact the project's PO and make sure they're aware of the urgency.
- Do PRs exist for these changes?
  - [ ] If so, have they been approved?

Ideally all of the PRs involved in breaking changes would be merged in the same day to avoid breaking the folio-testing environment.  Communication is paramount if that is to be achieved, especially as the number of intermodule and inter-team dependencies increase.

While it's helpful for reviewers to help identify potential problems, ensuring that it's safe to merge is ultimately the responsibility of the PR assignee.

### CODE REVIEW

File: src/OrderLinesList/OrderLinesList.test.js [hunk 1]
```diff
@@ -24,6 +24,9 @@ const mockLocalStorageFilters = {
 jest.mock('react-virtualized-auto-sizer', () => jest.fn(
   (props) => <div>{props.children({ width: 123 })}</div>,
 ));
+jest.mock('@folio/stripes-core/src/components/TitleManager/TitleManager', () => {
```
Comment (usavkov-epam) [inline src/OrderLinesList/OrderLinesList.test.js:27]: It should be mocked on the `stripes-acq-components` level in jest config. Please try to update deps and run tests without this code

File: src/OrdersList/OrdersList.test.js [hunk 2]
```diff
@@ -36,6 +36,9 @@ jest.mock('@folio/stripes/smart-components', () => ({
 jest.mock('@folio/stripes-components/lib/Commander', () => ({
   HasCommand: jest.fn(({ children }) => <div>{children}</div>),
 }));
+jest.mock('@folio/stripes-core/src/components/TitleManager/TitleManager', () => {
```
Comment (usavkov-epam) [inline src/OrdersList/OrdersList.test.js:39]: https://github.com/folio-org/ui-orders/pull/1568/files#r1514561379

File: src/OrderLinesList/OrderLinesList.test.js (hunk 1 above)
Comment (alisher-epam) [reply src/OrderLinesList/OrderLinesList.test.js:27]: Like that? https://github.com/folio-org/stripes-acq-components/commit/0686ef186079840b960cc85997f69b90673954df

Comment (usavkov-epam) [reply src/OrderLinesList/OrderLinesList.test.js:27]: Yes

**Review by usavkov-epam — APPROVED** [review]

**Review by NikitaSedyx — APPROVED** [review]

**Pull request merged**